MapboxNavigation/Locale.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+extension Locale {
+    static var preferredLocalLanguageCountryCode: String {
+        let bundleLanguage = Bundle.main.preferredLocalizations.first!
+        let bundleLanguages = bundleLanguage.components(separatedBy: "-")
+        
+        if bundleLanguages.count == 2 {
```

Comment:
It’s possible for the host application to have a localization whose locale code contains more than two components, for instance `zh-Hant-HK` for Traditional Chinese in Hong Kong versus `zh-Hant-TW` for the same in Taiwan.